storage/types.go

```diff
@@ -373,13 +373,14 @@ func (n *NodeID) Split(prefixBytes, suffixBits int) ([]byte, Suffix) {
 		panic(fmt.Sprintf("storage Split: %x(n.PrefixLenBits: %v - prefixBytes: %v *8) <= 0", n.Path, n.PrefixLenBits, prefixBytes))
 	}
 	suffixBytes := bytesForBits(b)
+	maskIndex := (b - 1) / 8
+	maskLowBits := (b-1)%8 + 1
+	path := a[prefixBytes : prefixBytes+suffixBytes]
```

Comment:
nit: Maybe rename this to `sfxPath` to make the diff a little clearer?

Reply:
Done.